cmd/askd/routes/routes.go

```diff
@@ -102,7 +102,12 @@ func routes(a *app.App) {
 	if publicKey != "" {
 		log.Dev("startup", "Init", "Initializing Auth")
 
-		authm, err := auth.Midware(publicKey)
+		authm, err := auth.Midware(publicKey, auth.MidwareOpts{
```

Comment:
Can we create the value above with the comments for readability. Then pass this value in? The less we embed values and functions during construction the better.